Heading/Heading.module.less

```diff
@@ -15,6 +23,17 @@
 		text-transform: @agate-heading-text-transform;
 		font-variant-ligatures: @agate-heading-font-ligatures;
 
+		.backButton {
+			.bg {
+				border: none;
+				background-color: transparent;
+			}
+
+			.client {
+				color: @agate-heading-text-color
+			}
```

Comment:
What is this color for?

Reply:
I think maybe Button update wasn't merged at that time and it didn't render the correct color for the icon of this back button. I honestly can't remember exactly. I will remove it.